desc/grid.py

```diff
@@ -1128,8 +1128,11 @@ def ocs(L):
         dt = np.asarray(dt)
         dimzern = r.size
 
-        z = np.linspace(0, 2 * np.pi / NFP, 2 * N + 1, endpoint=False)
-        dz = 2 * np.pi / z.size
+        # z = np.linspace(0, 2*np.pi/NFP , 2 * N + 1, endpoint=False)
+        # dz = 2 * np.pi / z.size
+
+        z = np.linspace(0, 1, 2*N + 1, endpoint=False)
```

Comment:
```suggestion
        z = np.linspace(0, 1, 2*N + 1, endpoint=True)
```

Reply:
In mirrors, cross-section 0 and 1 are different, we should probably keep them both.

Reply:
I think this will change some other logic within the code. I'm not too sure this is a good idea